overlord/devicestate/devicemgr.go

```diff
@@ -152,10 +167,10 @@ func (m *DeviceManager) ensureOperational() error {
 
 	if device.Brand == "" || device.Model == "" {
 		// need first-boot, loading of model assertion info
-		if release.OnClassic {
-			// TODO: are we going to have model assertions on classic or need will need to cheat here?
-			return nil
-		}
+
+		// TODO: on classic if seeded means there was no model
+		// use a fallback
+
 		// cannot proceed yet, once first boot is done these will be set
 		// and we can pick up from there
```

Comment:
Would you mind to merge all these comments into a single more prosaic comment that explains what we do now and what we need to do in the future. The first and last comments feel a bit redundant, and with a TODO in the middle it's a bit more confusing now.